framework/packages/abstract-sdk/src/apis/version_registry.rs

```diff
@@ -95,29 +92,34 @@ impl<'a, T: ModuleRegistryInterface> ModuleRegistry<'a, T> {
         infos: Vec<ModuleInfo>,
     ) -> AbstractSdkResult<Vec<ModuleResponse>> {
         let registry_addr = self.base.abstract_registry(self.deps)?.address;
-        let ModulesResponse { modules } = self.deps.querier.query(&wasm_smart_query(
-            registry_addr.into_string(),
-            &QueryMsg::Modules { infos },
-        )?)?;
+        let ModulesResponse { modules } = self
+            .deps
+            .querier
+            .query_wasm_smart(registry_addr.into_string(), &QueryMsg::Modules { infos })?;
```

Comment:
Could these queries get their own Error message? Should help in debugging if a module is missing.

Reply:
Nevermind, the smart query should return a good error anyway!